### DISCUSSION TITLE
language translation

### DISCUSSION THREAD

**Discussion by kassrose** [iptv-org/community#297] [Q&A] [answered]
Ok so i am trying to use the playlist that filters channels by country to watch a few channels on the usa based list and every channel i want to watch is either in spanish, german or some other language other than english and i cant figure out how to swap the languages while in the hypnotix iptv client. all the channels i am trying to watch should be in english as they are us based channels playing us made shows that origanally aired in the usa in english but i cant for the life of me get it to play in english.

**Comment by BellezaEmporium** [comment, marked as answer]
Let me split my answer here for a better comprehension.

1) Not all channels airing in the US are made for English people. There's some channels that are targeting certain diasporas or communities (eg Telemundo, VBS)
2) The database (up at https://iptv-org.github.io) is an excellent indicator of which channels exist in the US and which are the languages spoken. You just need to click on the name of the channel to have a detailed description of it.
